src/olympia/stats/models.py

```diff
@@ -166,7 +167,17 @@ class Meta:
         db_table = 'stats_contributions'
 
     def __unicode__(self):
-        return u'%s: %s' % (self.addon.name, self.amount)
+        return u'%s: %s %s' % (self.addon.name, self.amount, self.currency)
+
+    def __init__(self, *args, **kw):
+        super(Contribution, self).__init__(*args, **kw)
+        # TODO collect exchange rates from web service and store them somewhere
```

Comment:
I'm planning on pulling these from openexchangerates.org, on a daily cron job. But how to store them?